src/main/java/soot/jimple/internal/JLookupSwitchStmt.java

```diff
@@ -75,6 +76,21 @@ public Object clone() {
     return new JLookupSwitchStmt(getKey(), clonedLookupValues, getTargets(), getDefaultTarget());
   }
 
+  private String toSimpleString() {
+    final char endOfLine = ' ';
+    List<String> cases = lookupValues.stream()
+            .map((lookupValue) -> String.valueOf(lookupValue.value))
+            .distinct()
+            .collect(Collectors.toList());
+    if (getDefaultTarget() != null) {
+      cases.add("default");
+    }
+    return Jimple.LOOKUPSWITCH + "(" + keyBox.getValue().toString() + ')' + endOfLine
+              + "{" + endOfLine
+              + "cases:" + String.join(",", cases) + endOfLine
```

Comment:
please use `stream(). ... .collect(Collectors.joining(",")` directly, as the List of cases is not used elsewhere, removing the need to allocate a list